coaster-nn/src/frameworks/native/helper.rs

```diff
@@ -4,15 +4,19 @@ use crate::co;
 use crate::co::plugin::Error as PluginError;
 use crate::co::plugin::numeric_helpers::Float;
 use crate::co::frameworks::native::flatbox::FlatBox;
+use crate::{DirectionMode, RnnInputMode};
+use RnnNetworkMode;
 
 #[derive(Debug, Copy, Clone)]
 #[allow(missing_docs)]
 pub struct NormalizationConfig;
+
 #[derive(Debug, Clone)]
 #[allow(missing_docs)]
 pub struct PoolingConfig {
     pub window: Vec<i32>,
-    pub padding: Vec<i32>, //TODO: check datatype
+    pub padding: Vec<i32>,
+    //TODO: check datatype
```

Comment:
I think we can drop this comment, I am not sure why we should check this datatype anymore?

Reply:
Haven't done any work on CNN, so I'm not sure why we did it either.